File: Wizzi/Dtos/Sucursales/VerSucursalDto.cs
using Wizzi.Dtos.Localizaciones;

namespace Wizzi.Dtos.Sucursales
{
    public class VerSucursalDto
    {
        public string Codigo { get; set; }
        public string Nombre { get; set; }
        public string Direccion { get; set; }
        public bool EsMatriz { get; set; }
        public Localizacion localizacion { get; set; }
    }

    public class Localizacion
    {
        public VerPaisDto Pais { get; set; }
        public VerProvinciaDto Provincia { get; set; }
        public VerCantonDto Canton { get; set; }
        public VerParroquiaDto Parroquia { get; set; }
    }

}

File: Wizzi/ClientApp/src/dtos/Sucursales/verSucursalDto.ts
import { VerPaisDto, VerProvinciaDto, VerCantonDto, VerParroquiaDto } from "../Localizaciones";

export interface VerSucursalDto {
    codigo: string;
    nombre: string;
    direccion: string;
    esMatriz: boolean;
    localizacion: Localizacion;
}

export interface Localizacion {
    pais: VerPaisDto;
    provincia: VerProvinciaDto;
    canton: VerCantonDto;
    parroquia: VerParroquiaDto;
}